Makefile

```diff
@@ -46,17 +46,13 @@ lint-chart:
 	helm lint charts/ks-devops
 
 install-chart: lint-chart
-	helm install ks-ctl charts/ks-devops -n kubesphere-devops-system --set serviceAccount.create=true --create-namespace \
-		--set image.pullPolicy=Always \
-		--set jenkins.ksAuth.enabled=true
+	helm install ks-devops charts/ks-devops -n kubesphere-devops-system --create-namespace
```

Comment:
The default setting is to ensure that the service works properly. However the --set parameter is adjusted for custom installations.

Reply:
> I suggest we should keep those default value we set before.

The previous default was not to make the system work properly

Reply:
> The previous default was not to make the system work properly

What is the problem with the previous parameter? It would be great if you can post the error output or other log.